docs/source/installation.rst

```diff
@@ -0,0 +1,43 @@
+Installation Guide
+==================
+
+Quick Installation
+==================
```

Comment:
The headers quick installation, for developers, troubleshooting all should be at a level below "Installation Guide". Level 2 headers using `---------------------` instead.